android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -257,56 +255,85 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
       }
     });
 
-    return webView;
+    return wrapper;
   }
 
   @ReactProp(name = "javaScriptEnabled")
-  public void setJavaScriptEnabled(WebView view, boolean enabled) {
-    view.getSettings().setJavaScriptEnabled(enabled);
+  public void setJavaScriptEnabled(RNCWebView view, boolean enabled) {
+    view.ifPresent(webView -> webView.getSettings().setJavaScriptEnabled(enabled));
```

Comment:
this doesn't seem to be so bad with lambda expressions

Reply:
Agreed! I like this `ifPresent` API that you added 🙂